core/cfdefs/cfdef_standard.php

```diff
@@ -392,9 +397,10 @@ function cfdef_input_textarea( array $p_field_def, $p_custom_field_value ) {
  *
  * @param string $p_field_def          The custom field definition.
  * @param string $p_custom_field_value The custom field value to print.
+ * @param string $p_required          The "required" attribute to add to the field
  * @return void
  */
-function cfdef_input_date( $p_field_def, $p_custom_field_value ) {
+function cfdef_input_date( $p_field_def, $p_custom_field_value, $p_required = '' ) {
```

Comment:
Parameter is not used, shouldn't print_date_selection_set() be adapted to accept it ?
Considering the ongoing work on refactoring date fields, maybe this should be deferred, in that case adding a TODO here and in print_date_selection_set() as a reminder to eventually update this would be useful.